scripts/exrap_dump.sh

```diff
@@ -675,7 +676,7 @@ fi
 #  (default)
 
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_4} 1 msonet gpsipw
+$USHbufr_dump/bufr_dump_obs.sh $dumptime ${def_time_window_4} 1 msonet gpsipw
```

Comment:
This change is not required. The original $uscript_dump is defined in JOBSPROC_RAP_DUMP and can be retained. Please revert back to original code:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_1} 1 msonet gpsipw

Reply:
revised:
$ushscript_dump/bufr_dump_obs.sh $dumptime ${def_time_window_4} 1 msonet gpsipw